src/thunderbots/software/ai/navigator/obstacle/velocity_obstacle.cpp

```diff
@@ -0,0 +1,43 @@
+#include "ai/navigator/obstacle/velocity_obstacle.h"
+Polygon VelocityObstacle::getBoundaryPolygon(const Robot& robot,
+                                             double robot_radius_scaling,
+                                             double velocity_projection_scaling)
+{
+    double radius             = ROBOT_MAX_RADIUS_METERS * robot_radius_scaling;
+    Point velocity_projection = robot.velocity() * velocity_projection_scaling;
```

Comment:
I would recommend using the `norm()` function here to resize the vector. The multiplication operator doesn't quite do what you expect here.

Reply:
ok